wallet/src/main/AndroidManifest.xml

```diff
@@ -20,6 +45,44 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+
+        <activity
+            android:name=".PresentationActivity"
+            android:exported="true"
+            android:label="@string/app_name_presentation"
+            android:theme="@style/Theme.IdentityCredential">
+        </activity>
+
+        <service
+            android:name=".NfcEngagementHandler"
+            android:exported="true"
+            android:label="@string/nfc_engagement_service_desc"
+            android:permission="android.permission.BIND_NFC_SERVICE">
+            <intent-filter>
+                <action android:name="android.nfc.action.NDEF_DISCOVERED" />
+                <action android:name="android.nfc.cardemulation.action.HOST_APDU_SERVICE" />
+            </intent-filter>
+
+            <meta-data
+                android:name="android.nfc.cardemulation.host_apdu_service"
+                android:resource="@xml/nfc_engagement_apdu_service" />
+        </service>
+
+        <service
+            android:name=".NfcDataTransferHandler"
+            android:exported="true"
+            android:label="@string/nfc_data_transfer_service_desc"
+            android:permission="android.permission.BIND_NFC_SERVICE">
+            <intent-filter>
+                <action android:name="android.nfc.action.NDEF_DISCOVERED" />
+                <action android:name="android.nfc.cardemulation.action.HOST_APDU_SERVICE" />
+            </intent-filter>
+
+            <meta-data
+                android:name="android.nfc.cardemulation.host_apdu_service"
+                android:resource="@xml/nfc_data_transfer_apdu_service" />
+        </service>
+
```

Comment:
I don't think we need this <service> stanza since we're not implementing NFC data transfer (only BLE data transfer for now).